src/openmmio.py

```diff
@@ -1191,7 +1191,18 @@ def optimize(self, shot, crit=1e-4, disable_vsite=False, align=True, include_res
         # printcool_dictionary(energy_components(self.simulation), title='Energy component analysis before minimization, shot %i' % shot)
         # Minimize the energy.  Optimizer works best in "steps".
         for logc in np.linspace(0, np.log10(crit), steps):
-            self.simulation.minimizeEnergy(tolerance=10**logc*kilojoule/mole, maxIterations=10000)
+            self.simulation.minimizeEnergy(tolerance=10**logc*kilojoule/mole, maxIterations=100000)
+        # check if energy minimization is successful
+        # try 10 times as openmm minimizer is not very stable at the tolerance
```

Comment:
Oh wow. That's very unfortunate. I'm not very experienced with OpenMM, but I did find cases where the `Reference` platform gave me inconsistent results, while the `OpenCL` platform was fully consistent between reruns of the same system. If these retries add a significant time cost increase, hard-coding the platforms may help us in the run-up to release 1. It would, however, make the code less general.

Reply:
I agree it's worth revisiting the problem later and probably raise an issue for OpenMM. I think trying to re-run the minimizeEnergy here is working for now.